packages/devtools_app/lib/src/screens/network/network_request_inspector.dart

```diff
@@ -27,8 +27,6 @@ class NetworkRequestInspector extends StatelessWidget {
   @visibleForTesting
   static const headersTabKey = Key(_headersTabTitle);
   @visibleForTesting
-  static const requestTabKey = Key(_requestTabTitle);
```

Comment:
fine to remove this, but can you add a TODO to clean to remove these keys and find widgets by text  in our tests? you can assign to me

`// TODO(kenz): remove these keys and use a text finder to lookup widgets in test.`